assets/i18n/en.json

```diff
@@ -9,7 +9,7 @@
     "I18N_ABOUT_FOUNDATION_PAGE_HEADING": "The Oppia Foundation",
     "I18N_ABOUT_FOUNDATION_PAGE_HEADING_CONTENT": "Building a community to provide quality education to those who lack access to it",
     "I18N_ABOUT_FOUNDATION_PAGE_MILLIONS_OF": "Millions of children are not receiving the education they deserve.",
-    "I18N_ABOUT_FOUNDATION_PAGE_MILLIONS_OF_CONTENT": "Of the <a href=\"http://uis.unesco.org/en/news/world-poverty-could-be-cut-half-if-all-adults-completed-secondary-education\" class=\"inline-links\">61 million children</a> worldwide who cannot attend primary school, 17 million will never step foot into a school if trends continue. Unfortunately, <a href=\"http://uis.unesco.org/sites/default/files/documents/fs46-more-than-half-children-not-learning-en-2017.pdf\" class=\"inline-links\">even those who are in school</a> may not be getting the effective education they need.",
+    "I18N_ABOUT_FOUNDATION_PAGE_MILLIONS_OF_CONTENT": "Of the <a href=\"http://uis.unesco.org/en/news/world-poverty-could-be-cut-half-if-all-adults-completed-secondary-education\" class=\"inline-links\">61 million children</a> worldwide who cannot attend primary school, 17 million will never step foot into a school if trends continue. Unfortunately, <a href=\"https://uis.unesco.org/sites/default/files/documents/fs46-more-than-half-children-not-learning-en-2017.pdf\" class=\"inline-links e2e-external-pdf-link\">even those who are in school</a> may not be getting the effective education they need.",
```

Comment:
Ah -- no, don't do this, we cannot do it if it affects translations I'm afraid. It adds too much random stuff for translators to translate, and on balance I prefer having the headless problem to messing with the translations.

I have another suggestion to mitigate -- have a function in the acceptance tests that verifies external PDF links, and have that function skip execution if run in headless mode (add a comment to explain, too).

Reply:
Maybe make a function in puppeteer-utils.ts to check if it's running in headless mode, and then call this function before trying to click the href. I believe this issue should be able to fix without changing any files in assets/i18n.

Reply:
@seanlip @imchristie, I understand your concerns and fine with another approach suggested. However, I’m wondering if the changes I’ve made truly affect translations. For instance, I added an ‘s’ to ‘http’, but I could avoid this by removing the ‘s’ from the expected URL. However, the URL remains same across available translations. 

Additionally, I’ve added a class, and classes remains consistent across all translations. I believe I need to add this class to all available translations.

Could you please confirm if my understanding is correct or am I missing something?

Reply:
They do make translations more complex. Translations are done manually on translatewiki and I keep having to filter them occasionally because they get the html wrong. Let's try to reduce that occurrence.

Reply:
@seanlip, I’ve made a note. Thank you for your patience in explaining things.